src/appMain/hmi_capabilities.json

```diff
@@ -482,201 +482,241 @@
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"RIGHT_TURN_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"HAZARD_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"AMBIENT_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"OVERHEAD_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"READING_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"TRUNK_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"EXTERIOR_FRONT_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"EXTERIOR_REAR_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
                                  "densityAvailable":true,
                                  "name":"EXTERIOR_LEFT_LIGHTS",
                                  "sRGBColorSpaceAvailable":true
                               },
                               {
+                                 "statusAvailable":true,
```

Comment:
The new light names shall be listed in this hmi_capabilities.json, an example is
```xml
 {  
  "statusAvailable":false,  
  "densityAvailable":false,
  "name":"REAR_CARGO_LIGHTS",   
   "sRGBColorSpaceAvailable":false
 }, 
``` 
and similar contents for other new added light names
REAR_TRUCK_BED_LIGHTS
REAR_TRAILER_LIGHTS
LEFT_SPOT_LIGHTS
RIGHT_SPOT_LIGHTS
LEFT_PUDDLE_LIGHTS
RIGHT_PUDDLE_LIGHTS

and 
```xml
 {  
  "statusAvailable":true,  
  "densityAvailable":false,
  "name":"EXTERIOR_ALL_LIGHTS",   
   "sRGBColorSpaceAvailable":false
 }, 
```

Reply:
@yang1070 we will change this capabilities on HMI side

Reply:
need to change this json file too

Reply:
@yang1070 please look at 2eb2d0b2b0290c78ccc648274e7f88bf465d1e82